pages/documentation.html

```diff
@@ -12,7 +12,10 @@
 <ul>
 
 <li>the <a href="/distrib/current/refman/">Reference Manual</a>, which
-is the complete, authoritative source of documentation for Coq;</li>
+is the complete, authoritative source of documentation for Coq; it
+contains
+the <a href="/distrib/current/refman/changes.html">changelog</a>,
+which we recommend you read when you upgrade your version of Coq;</li>
```

Comment:
```suggestion
is the authoritative source of documentation for Coq.  It
contains
a <a href="/distrib/current/refman/changes.html">changelog</a>
describing updates to Coq,
which we recommend you read when you upgrade Coq;</li>
```

I hesitate to say the doc is complete.